src/TmsIntegrationExample.Services/TmsGatewayApi/Model/Accessorial.cs

```diff
@@ -0,0 +1,24 @@
+﻿// <copyright file="Accessorial.cs" company="Engaged Technologies">
+//      Copyright (c) Engaged Technologies. All rights reserved.
+//  </copyright>
+// <summary>
+//     Model for an Accessorial
+// </summary>
+
+namespace TmsIntegrationExample.Services.TmsGatewayApi.Model
+{
+    public class Accessorial
+    {
+        /// <summary>
+        /// Gets or sets the accessorial code.
+        /// </summary>
+        /// <value>The accessorial code.</value>
```

Comment:
We shouldn't need to include swagger doc related comments in this example if TmsGateway exposes swagger

Reply:
do we need to remove the comments for all models classes or just this one?